package.json

```diff
@@ -1,20 +1,25 @@
 {
   "name": "recipe-app",
   "version": "0.1.0",
+  "proxy": "http://localhost:8080",
```

Comment:
don't mark anything as localhost, because this won't always work when you're running in production

Reply:
done